src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.ts

```diff
@@ -1,42 +1,124 @@
+import { Store } from '@ngrx/store';
+import { ControlTypeEnum } from 'src/hearings/models/hearings.enum';
+import { RefDataModel } from '../../../models/refData.model';
+
+import { HearingJudgeNamesListComponent } from '../../../../hearings/components';
+import { ACTION, HearingPanelSelectionEnum } from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
 import * as fromHearingStore from '../../../store';
-import {RequestHearingPageFlow} from '../request-hearing.page.flow';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
 
 @Component({
   selector: 'exui-hearing-panel',
   templateUrl: './hearing-panel.component.html',
 })
 export class HearingPanelComponent extends RequestHearingPageFlow implements OnInit, AfterViewInit, OnDestroy {
+  public multiSelectArray: FormArray;
   public panelJudgeForm: FormGroup;
   public validationErrors: { id: string, message: string }[] = [];
   public includedJudgeList: Person[] = [];
   public excludedJudgeList: Person[] = [];
   public panelSelection: string;
+  public multiSelection: RefDataModel[];
   public panelSelectionError: string;
+  public configLevels: { level: number, controlType: ControlTypeEnum }[];
   @ViewChild('includedJudge') public includedJudge: HearingJudgeNamesListComponent;
   @ViewChild('excludedJudge') public excludedJudge: HearingJudgeNamesListComponent;
 
-  constructor(protected readonly hearingStore: Store<fromHearingStore.State>,
-              protected readonly hearingsService: HearingsService,
-              private readonly formBuilder: FormBuilder,
-              protected readonly route: ActivatedRoute) {
-    super(hearingStore, hearingsService, route);
+  constructor(
+    protected readonly hearingStore: Store<fromHearingStore.State>,
+    protected readonly hearingsService: HearingsService,
+    protected readonly route: ActivatedRoute,
+    private readonly formBuilder: FormBuilder) {
+    super(hearingStore, hearingsService);
+    this.multiSelection = this.route.snapshot.data.listOffValues;
+    this.configLevels = [
+      {
+        controlType: ControlTypeEnum.CHECK_BOX,
+        level: 1,
+      },
+      {
+        controlType: ControlTypeEnum.SELECT,
+        level: 2,
+      }
+    ];
   }
 
   public ngOnInit(): void {
     this.initForm();
   }
 
+  public arrayValidator(min: number) {
+    return (c: AbstractControl): { [key: string]: any } => {
+      if (c.value.length >= min) {
+        return null;
+      }
+
+      return null; // { minLengthArray: { valid: false } };
+    };
+  }
+
   public initForm(): void {
     this.panelJudgeForm = this.formBuilder.group({
       specificPanel: ['', Validators.required],
+      multiSelect: this.formBuilder.array([], this.arrayValidator(1))
+    });
+
+    this.panelJudgeForm.controls.multiSelect = this.convertRefDataModelToArray(this.multiSelection);
+  }
+
+  public convertArrayToRefDataMode(array: FormArray): RefDataModel[] {
```

Comment:
Minor spelling mistake. Should be `convertArrayToRefDataModel ` not `convertArrayToRefDataMode`